src/transformState.js

```diff
@@ -5,7 +5,31 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  let modifiedState = Object.assign(state);
+
+  actions.forEach(action => {
+    switch (action.type) {
+      case 'addProperties':
+        modifiedState = Object.assign(modifiedState, action.extraData);
```

Comment:
You can change here state directly
```suggestion
        Object.assign(state, action.extraData);
```